docs/pages/Cardinal/API-Reference/World.mdx

```diff
@@ -0,0 +1,529 @@
+| ns        | string  | The namespace to be set for World. |
+
+
+## Methods
+
+### AddSystems
+
+`AddSystems` adds one or more systems to the `World`. Systems are executed in the order of which they were added to the world.
+
+```go
+func (w *World) AddSystems(s ...System)
+```
+
+#### Example
+
+```go
+package main
+
+import (
+    "pkg.world.dev/world-engine/cardinal/ecs"
+
+    "github.com/my-username/my-world-engine-project/systems"
+)
+
+func main() {
+    // ... world setup ...
+
+    world.AddSystems(systems.MoveSystem, systems.HealthRegenSystem, systems.AttackSystem)
+}
+```
+
+#### Parameters
+
+| Parameter | Type         | Description                                             |
+|-----------|--------------|---------------------------------------------------------|
+| s         | ...System    | Variadic parameter for systems to be added to the World. |
+
+
+### RegisterComponents
+
+`RegisterComponents` registers one or more components to the `World`. Upon registration, components are assigned an ID. IDs are assigned incrementally, starting from 0, in the order in which they were passed to the method.
+
+
+<Callout type={"warning"}>
+    `RegisterComponents` can be only be called once. Subsequent calls to the method will return an error.
+</Callout>
+
+
+```go
+func (w *World) RegisterComponents(components ...component.IComponentType) error
+```
+#### Example
+
+```go
+package main
+
+import (
+	"log"
+
+    "pkg.world.dev/world-engine/cardinal/ecs"
+
+    "github.com/my-username/my-world-engine-project/components"
+)
+
+func main() {
+    // ... world setup ...
+
+    err := world.RegisterComponents(
+		components.LocationComponent,
+		components.AttackPowerComponent,
+		components.HealthComponent,
+	)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+```
+
+#### Parameters
+
+| Parameter    | Type                    | Description                                                    |
+|--------------|-------------------------|----------------------------------------------------------------|
+| components   | ...component.IComponentType | Variadic parameter for components to be registered with the World. |
+
+#### Return Value
+
+| Type | Description                                                                  |
+|-------------|------------------------------------------------------------------------------|
+| error       | An error indicating any issues that occurred during the component registration. |
+
+
+### RegisterReads
+
+`RegisterReads` registers the reads in the `World`. This allows the `Read` endpoints to be automatically generated.
+
+```go
+func (w *World) RegisterReads(reads ...IRead) error
+```
+#### Example
+
+```go
+package main
+
+import (
+	"log"
+
+    "pkg.world.dev/world-engine/cardinal/ecs"
+
+    "github.com/my-username/my-world-engine-project/reads"
+)
+
+func main() {
+    // ... world setup ...
+
+    err := world.RegisterReads(
+		reads.ReadLocation,
```

Comment:
package name in starter-game-template is singular `read` would like to keep it consistent

Reply:
[package consistency:](https://github.com/Argus-Labs/world-engine/pull/200/commits/3ace678c1ac3dfe79a376c766e89708b80e22404)